Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/MSFT_AADRoleSetting.psm1

```diff
@@ -1379,7 +1380,7 @@ function Export-TargetResource
     try
     {
         $Script:ExportMode = $true
-        [array] $Script:exportedInstances = Get-MgBetaRoleManagementDirectoryRoleDefinition -ErrorAction Stop
+        [array] $Script:exportedInstances = Get-MgBetaRoleManagementDirectoryRoleDefinition -Filter $Filter -Sort DisplayName -ErrorAction Stop
```

Comment:
Not sure what will happen, if filter is empty? Does it make sense to add the Filter only when it is set?

Reply:
i've done a recheck and Get-MgBetaRoleManagementDirectoryRoleDefinition with empty Filter returns the same results as without